src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,29 +1,52 @@
-export const TodoFilter = () => (
+import { Filter } from '../../types/Filter';
+
+type Props = {
+  inputValue: string;
+  handleSelectedOption: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+  handleQueryChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  resetFilterParams: () => void;
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  inputValue,
+  handleSelectedOption,
+  handleQueryChange,
+  resetFilterParams,
+}) => (
   <form className="field has-addons">
     <p className="control">
       <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
+        <select data-cy="statusSelect" onChange={handleSelectedOption}>
+          <option value="all">{Filter.ALL}</option>
+          <option value="active">{Filter.ACTIVE}</option>
+          <option value="completed">{Filter.COMPLETED}</option>
```

Comment:
The `option` values should be string literals ('all', 'active', 'completed') instead of referencing `Filter.ALL`, `Filter.ACTIVE`, and `Filter.COMPLETED`. This ensures that the `select` element's value matches the expected string values in the `handleSelectedOption` function in `App.tsx`.